bears/python/VultureBear.py

```diff
@@ -1,25 +1,45 @@
-import re
 from shutil import which
-
+from vulture import Item, Vulture
 from coalib.bears.GlobalBear import GlobalBear
 from dependency_management.requirements.PipRequirement import PipRequirement
-from coalib.misc.Shell import run_shell_command
 from coalib.results.Result import Result
 
 
+class Unused(Item):
+
+    def __new__(cls, item, message):
+        item.message = message
+        return item
+
+
+def retrieve_unused(filenames):
+
```

Comment:
Needs a docstring, and possible this function should be private by prefixing name with `_`